HW2Question2.cpp

```diff
@@ -0,0 +1,91 @@
+//
+//  HW2Question2.cpp
+//  Codu
+//
+//  Created by Farida Eid on 5/27/17.
+//  Copyright © 2017 Farida Eid. All rights reserved.
+//
+
+#include <iostream>
+#include <vector>
+
+using namespace std;
+
+struct treeNode
+{
+    int value;
+    treeNode *left;
+    treeNode *right;
+    
+    treeNode(int v):value(v), left(NULL), right(NULL){}
+    
+};
+
+// find path of each node and checks if the node is in the tree or not
+
+bool findPath(treeNode * head, int key, vector<int>& answer)
```

Comment:
Same as with previous problem, please use a pointer for the parameter you want to modify.